metadata.json

```diff
@@ -16,6 +16,10 @@
       "name": "puppetlabs/augeas_core",
       "version_requirement": ">= 1.0.0 < 2.0.0"
     },
+    {
+      "name": "puppetlabs/yumrepo_core",
```

Comment:
the core repos should not be in the metadata.json, because puppet AIO packages vendor them (I need to remove the augeas_core as well)

Reply:
I removed this